addons/website_slides/models/slide_slide.py

```diff
@@ -401,29 +503,29 @@ def create(self, values):
             # 'website_published' is handled by mixin
             values['date_published'] = False
 
-        if values.get('slide_category') == 'infographic' and not values.get('image_1920'):
```

Comment:
Side note: could be great to make create multi-enabled (but not mandatory, if you feel this is going to take too much time, can be done afterwards).